Makefile

```diff
@@ -4,20 +4,25 @@ MAKEFLAGS += --no-builtin-rules
 SHELL = /bin/bash
 .SHELLFLAGS = -o pipefail -c
 
-# Build options can either be changed by modifying the makefile, or by building with 'make SETTING=value'
-# It is also possible to override default settings in a file called .make_options.mk with 'SETTING=value'.
+#### Build options ####
+
+# The build options below can be changed by modifying the Makefile, or by appending 'SETTING=value' to all
```

Comment:
maybe 
"The build options can be changed by modifying them below, or by appending..."

saying "by modifying the makefile" when we are already in the makefile may be confusing